### DISCUSSION TITLE
Is possible to commit and push (+ pull) when saving the file?

### DISCUSSION THREAD

**Discussion by ignaciolg** [Vinzent03/obsidian-git#385] [Q&A]
Hi, 

First, thank you for this awesome plugin. I'm using it a lot (and doing weird stuff like running git hooks when committing code because why not)

My question is to know if it is possible `add + commit + push` automatically when saving a file. 

If it is not possible, do you think it can be implemented? 

Sorry, I'm not used to the Obvisidan plugin system yet so I can not help with  PR.

**Comment by Vinzent03** [comment]
Thank you. There is an `Auto backup after file change` setting. You can achieve what you want by enabling the option and setting the interval to 1.
